Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.5-alpine
+
+ENV RUBYOPT "rubygems"
+
+COPY Gemfile /usr/src/CeWl/
+WORKDIR /usr/src/CeWl
+
+RUN set -ex \
+    && apk add  --no-cache --virtual .build-deps build-base \
+    && gem install bundler \
+    && bundle install \
+    && apk del .build-deps
+
+COPY . /usr/src/CeWL
```

Comment:
Do we really want to copy the whole contents of the github repo into /usr/src/CeWL.
 a minimalist approach would be better.

Reply:
Yes, good idea. We should add a ``.dockerignore`` file with the line ``.git`` then.

Reply:
Done ! Thanks